shared/profile/wot-author/index.tsx

```diff
@@ -188,6 +194,79 @@ export const Question2Wrapper = (
   )
 }
 
+export const ReviewWrapper = (
+  props: Container.RouteProps<{
+    sigID: string // sigID of the vouch.
+  }>
+) => {
+  const sigID = Container.getRouteProps(props, 'sigID', '')
+  const dispatch = Container.useDispatch()
+  const voucheeUsername = Container.useSelector(state => state.config.username)
+  const details = Container.useSelector(state => Tracker2Constants.getDetails(state, voucheeUsername))
+  const wotEntry = (details.webOfTrustEntries || []).find(entry => entry.proofID === sigID)
```

Comment:
prefer to just use `details.webOfTrustEntries?.find(` instead of using the empty array